src/DocumentsRepository/Traits/SPARQLDocumentsRepositoryTrait.ts

```diff
@@ -51,6 +51,9 @@ export const SPARQLDocumentsRepositoryTrait:SPARQLDocumentsRepositoryTraitFactor
 			if( ! this.context.registry.inScope( uri, true ) ) return Promise.reject( new IllegalArgumentError( `"${ uri }" is out of scope.` ) );
 			const url:string = this.context.getObjectSchema().resolveURI( uri, { base: true } );
 
+			requestOptions = requestOptions ? requestOptions : {};
+			RequestUtils.setAcceptHeader( "application/ld+json; q=0.9", requestOptions! );
```

Comment:
If this line is seen without context it may be difficult to make sense out of it. Please add a comment above it to explain why the header needs to be added. Each time you write new code please ask yourself if you'll be able to understand why that code is doing what it does without remembering why it does it. If you doubt you'll be able to, add a comment explaining the "why".